pallets/api/src/messaging/test_utils.rs

```diff
@@ -26,10 +26,10 @@ pub fn ismp_post_request() -> PostRequest {
 		source: StateMachine::Polkadot(2000),
 		dest: StateMachine::Polkadot(2001),
 		nonce: 100u64,
-		from: vec![],
-		to: vec![],
+		from: [1u8].repeat(1000),
+		to: [1u8].repeat(1000),
 		timeout_timestamp: 10000,
-		body: vec![],
+		body: [1u8].repeat(1000),
```

Comment:
Similarly, does this one make sense to bound by the benchmarks variables as well? Do we have any config item for the "body" of the Post?